### ISSUE TITLE
Add option to make scrollbars always visible for long comments

### ISSUE THREAD

**Issue by sfederici** [ScratchAddons/ScratchAddons#6552]
### Why this would be helpful

My students add comments to scripts to answer questions. With the current options in the "Better editor comments" addon, I don't know if what I see is the full answer or if there is more. So I have to click the comment and scroll to see if what I see is the full answer.

### My suggestion

Add an option to "Better editor comments" addon to make scrollbars always visible when the comment is longer than the visible space

**Comment by Samq64** [comment]
You mean an option to prevent the scrollbars from auto-hiding, right? Because there's already an option to add them.

**Comment by sfederici** [comment]
Yes. I don't want the scrollbar to auto-hide for comments that are not fully visible, that is they are longer than space available in the post-it. But I want the scrollbar to auto-hide for comments that are completely visibile in the post-it. So that I know when it is necessary that I go to a comment and scroll it down.

**Comment by sfederici** [comment]
> Is this issue specific to macOS?

Yes, as far as I understand, specific of the combination macOS + Firefox. Indeed, I keep seeing scrollbars in macOS+Chrome and Win10+Chrome/Firefox.

**Comment by DNin01** [comment]
> > Is this issue specific to macOS?
> 
> Yes, as far as I understand, specific of the combination macOS + Firefox. Indeed, I keep seeing scrollbars in macOS+Chrome and Win10+Chrome/Firefox.

Thanks for the info. Yes, I just tested it, the scroll bars show up in Edge 116 on Windows 11.

It would be great if there was a solution for this, but I don't know if we want to override user preferences - I believe hidden scroll bars is a setting in macOS somewhere. Maybe instead we can just partially reveal the next line (cut it off) so the user can see parts of the next line but has to scroll to read it.
![Scrollable comment on Scratch with third line cut off](https://github.com/ScratchAddons/ScratchAddons/assets/106490990/48815ed6-1f3e-4536-b5aa-4d0e1909e23f)

**Comment by DNin01** [comment]
Oh wait, that's not the right addon! Whoops! But the same idea still applies. Although I'm not sure there's a way to make the next line partially visible on editor comments since ideally they should stay the same size.

**Comment by sfederici** [comment]
Thanks for letting me notice this. I confirm that the scrollbars can be setup as always visible in macos. So, even in firefox, they can be clearly seen when the text overflows.

I like your idea, so that even if the user keep using the standard settings they can clearly see if the text of the comments is overflowing.
